FordLabs/peoplemover
# Remove focus ring javascript logic and replace with css

## Issue
Resolves N/A

## What was done
There was a desire for everything link/button, etc to have a focus border around it when tabbing through the app, but not when clicking. This was previously handled with a window listener & a class. This PR removes that logic and replaces it with pure css.

## How to test
1. Tab through the app and ensure that a blue outline shows up where the focus state is.
2. Click around to the same elements and ensure that there is no focus border when clicking.
